skyplane/compute/gcp/gcp_pricing.py

```diff
@@ -0,0 +1,54 @@
+class GCPPricing:
+    @staticmethod
+    def get_transfer_cost(src_key, dst_key, premium_tier=True):
+        src_provider, src = src_key.split(":")
+        dst_provider, dst = dst_key.split(":")
+        assert src_provider == "gcp"
+        src_continent, src_region, src_zone = src.split("-")
+        if dst_provider == "gcp":
+            dst_continent, dst_region, dst_zone = dst.split("-")
+            if src_continent == dst_continent and src_region == dst_region and src_zone == dst_zone:
+                return 0.0
+            elif src_continent == dst_continent and src_region == dst_region:
```

Comment:
Could we load this from `profiles/costs.csv` instead? Might be cleaner and easier to maintain.

Reply:
Additionally, it would be great to reference the sources for these prices in some documentation. Perhaps a `pricing.md`? Otherwise, it will be hard to track when prices do (inevitably) change.

Reply:
I created an issue to track this: https://github.com/skyplane-project/skyplane/issues/666

Unfortunately, it's not easy to get egress pricing information from GCP so I had to manually collect this. In the future, it would be good to scrape this, but till then I think this logic is cleaner.